keps/NNNN-kep-template/kep.yaml

```diff
@@ -14,9 +14,14 @@ reviewers:
 approvers:
   - TBD
   - "@oscar.doe"
-prr-approvers:
-  - TBD
-  - "@bob.doe"
+
+##### WARNING !!! ######
+# prr-approvers has been moved to its own location
+# You should create your own in keps/prod-readiness
+# Please make a copy of keps/prod-readiness/template/nnnn.yaml
+# to keps/prod-readiness/sig-xxxxx/00000.yaml (replace with kep number)
+#prr-approvers:
+
```

Comment:
I'm not sure if we have stopped reading this value of `prr-approver` in our tooling. But the warning does make sense as an added caution.

Reply:
yes we did :D 

If you take a look into recent KEPs (like sig-cli/2386) there's no prr-approvers anymore. I sort of remember @johnbelamaric or someone else cleaning this from old KEPs as well :)

Reply:
Cool! This makes sense then. :)